src/auto-evo/simulation/food_source/FloatingChunkFoodSource.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+
+public class FloatingChunkFoodSource : FoodSource
+{
+    private readonly Compound glucose = SimulationParameters.Instance.GetCompound("glucose");
+
+    private Patch patch;
+    private BiomeConditions biomeConditions;
+    private float totalEnergy;
+    private float chunkSize;
+
+    public FloatingChunkFoodSource(Patch patch)
+    {
+        this.patch = patch;
+        biomeConditions = patch.Biome;
+
+        if (patch.Biome.Chunks.ContainsKey("marineSnow"))
+        {
+            ChunkConfiguration chunk = patch.Biome.Chunks["marineSnow"];
+            chunkSize = chunk.Size;
+            totalEnergy = chunk.Compounds[glucose].Amount * Constants.AUTO_EVO_CHUNK_ENERGY_AMOUNT;
+        }
+    }
+
+    public override float FitnessScore(Species species)
+    {
+        var microbeSpecies = (MicrobeSpecies)species;
+
+        var predatorSize = microbeSpecies.Organelles.Organelles.Sum(organelle => organelle.Definition.HexCount);
```

Comment:
It's not really a predator here, right? So `scavenger`?
Also this "engulf size" calculation is quite in many places, would be probably very handy to directly have this in MicrobeSpecies, as I would guess that someone somewhere will forget to take the IsBacteria flag into account.